lightbulb/checks.py

```diff
@@ -138,7 +138,9 @@ def _get_missing_perms(permissions: hikari.Permissions, roles: typing.Sequence[h
     for perm in permissions:
         if perm is hikari.Permissions.ADMINISTRATOR:
             return hikari.Permissions.NONE
-        if not any(role.permissions & perm for role in roles):
+
+        # Now it check if the role has admin. If it has admin, it doesn't count it as missing perm
+        if not any(role.permissions & perm or role.permissions & hikari.Permissions.ADMINISTRATOR for role in roles):
             missing_perms |= perm
     return missing_perms
```

Comment:
```suggestion
def _get_missing_perms(required_permissions: hikari.Permissions, roles: typing.Sequence[hikari.Role]) -> hikari.Permissions:
    missing_perms = hikari.Permissions.NONE
    user_permissions = set([role.permissions for role in roles])

    if hikari.Permissions.ADMINISTRATOR in user_permissions:
        return hikari.Permissions.None

    for required_permission in required_permissions:
        if required_permission not in user_permissions:
            missing_perms |= perm

    return missing_perms
```

This bug was caused by confusion in the naming, this was actually allowing anybody to execute a command that required the `hikari.Permissions.ADMINISTRATOR` permission

*This suggestion goes up to line 136, but GitHub wont let me select that up*